BUILD

```diff
@@ -17,36 +17,13 @@
 # under the License.
 #
 
-load("@io_bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
+load("@bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 
 exports_files([
-    "bazelbuild_rules_python-export-requirements-bzl-for-stardoc.patch",
     "bazelbuild_rules_pkg-fix-tarfile-format.patch",
 ])
 
-# Stardoc is unable to generate documentation unless it can
-# load files that our rule files depends on via load(...)
-# statements.
-# This means it needs to have them accessible within the
-# sandbox, which it can only do if it depends on the files
-# as source.
-# https://github.com/bazelbuild/skydoc/issues/166
-bzl_library(
```

Comment:
We no longer need this hack, since most bazel dependencies these days seem to either expose their own `bzl_library` to support stardoc using `bzl_library.deps`, or have an export of all files in the repository to allow others to use them in `bzl_library.srcs`